### PR TITLE
UX: Combine closed and archived icon.

### PR DESCRIPTION
**Pull request by tgxworld**
meta: https://meta.discourse.org/t/double-locked-topic/32712/3

### CODE REVIEW

File: app/assets/javascripts/discourse/components/topic-status.js.es6 [hunk 1]
```diff
@@ -28,21 +28,30 @@ export default Ember.Component.extend(StringBuffer, {
...
-      const title = Handlebars.Utils.escapeExpression(I18n.t("topic_statuses." + key + ".help")),
+    const renderIcon = function(name, key, actionable) {
+      const title = Handlebars.Utils.escapeExpression(I18n.t(`topic_statuses.${key}.help`)),
             startTag = actionable ? "a href" : "span",
             endTag = actionable ? "a" : "span",
             iconArgs = key === 'unpinned' ? { 'class': 'unpinned' } : null,
             icon = iconHTML(name, iconArgs);
 
-      buffer.push("<" + startTag + " title='" + title + "' class='topic-status'>" + icon + "</" + endTag + ">");
+      buffer.push(`<${startTag} title='${title}' class='topic-status'>${icon}</${endTag}>`);
```
Comment (tgxworld) [inline app/assets/javascripts/discourse/components/topic-status.js.es6]: Are we a fan of template strings?

**Comment by SamSaffron** [comment]
cool, looks good, we do need to figure out a clean way of centralizing the rendering here

File: app/assets/javascripts/discourse/components/topic-status.js.es6 (hunk 1 above)
Comment (ZogStriP) [inline app/assets/javascripts/discourse/components/topic-status.js.es6]: Yes we are :+1:

Comment (tgxworld) [inline app/assets/javascripts/discourse/components/topic-status.js.es6]: Awesome!

**Pull request merged**